src/uct/sm/scopy/base/scopy_iface.c

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#include <uct/sm/base/sm_iface.h>
+#include <uct/sm/scopy/base/scopy_iface.h>
+
+
+ucs_config_field_t uct_scopy_iface_config_table[] = {
+    {"SM_", "", NULL,
+     ucs_offsetof(uct_scopy_iface_config_t, super),
+     UCS_CONFIG_TYPE_TABLE(uct_sm_iface_config_table)},
+
+    {"MAX_IOV", "16",
```

Comment:
can it be common with `AM` as well? and then defined for all sms?

Reply:
we don't have AM Zcopy in MM.
So, I guess it's ok to have it defined only for SCOPY. Then, when AM Zcopy will be added we could move it to SM iface configuration

Reply:
My main concern was that the name does not reflect this is only rma (put/get) related config. But, right, no need to change now